pkg/assembler/graphql/schema/package.graphql

```diff
@@ -153,6 +153,18 @@ input PackageQualifierInput {
   value: String
 }
 
+"""
+PkgNameSpec is used for IsDependency to input dependent packages. This is different from PkgSpec
+as the IsDependency attestation should only be allowed to be made to the packageName Node and not the
+packageVersion node. Versions will be handled by the version_range in the IsDependency attestation node.
+"""
+input PkgNameSpec {
+  type: String
+  namespace: String
+  name: String
+}
+
+
```

Comment:
Can we move this in the same file where it gets used for the first time? Or, do you think we'll need to use it in multiple places, not just for dependencies?

Reply:
yea I was thinking of keeping with the other package inputs but I can move it to where it's used. I don't see it being used elsewhere.